android/app/src/main/python/electroncash_gui/android/console.py

```diff
@@ -3,9 +3,10 @@
 from code import InteractiveConsole
 import json
 import os
-from os.path import exists, join
+from os.path import exists, join, dirname
 import pkgutil
 import unittest
+from shutil import copyfile
 
```

Comment:
I like to keep standard library imports in alphabetical order as well, to make them easier to search through visually.